core/app/modules/import-export/assets/js/pages/import/import-process/import-process.js

```diff
@@ -22,6 +22,9 @@ export default function ImportProcess() {
 
 			if ( fileURL || context.data.fileResponse ) {
 				if ( fileURL ) {
+					fileURL[ 0 ] = decodeURIComponent( fileURL[ 0 ] );
```

Comment:
Why do you need to decode also `fileURL[ 0 ]`?

Reply:
Done